test/test_muscles.cpp

```diff
@@ -157,15 +159,22 @@ TEST(HillType, unitTest)
             model.muscleGroup(muscleGroupForHillType).muscle(
                 muscleForHillType));
         biorbd::rigidbody::GeneralizedCoordinates Q(model);
+        biorbd::rigidbody::GeneralizedVelocity QDot(model);
         Q = Q.setOnes() / 10;
-        hillType.updateOrientations(model, Q);
+        QDot = QDot.setOnes() / 10;
+        model.updateMuscles(Q, 2);
+        hillType.updateOrientations(model, Q, QDot);
         static double activationEmgForHillTypeTest(1.0);
         biorbd::muscles::StateDynamics emg(0, activationEmgForHillTypeTest);
 
-        EXPECT_EQ(hillType.FlCE(emg), 0.9928132);
+        EXPECT_NEAR(hillType.FlCE(emg), 0.67988981401208015, requiredPrecision);
+        EXPECT_NEAR(hillType.FlPE(), 0, requiredPrecision);
+        EXPECT_NEAR(hillType.FvCE(), 1.000886825333013, requiredPrecision);
+        EXPECT_NEAR(hillType.damping(), 0.00019534599393617336, requiredPrecision);
+        EXPECT_NEAR(hillType.force(emg), 424.95358302550062, requiredPrecision); //Fonctionne dans biorbd_new mais pas biorbd_ref?
```

Comment:
Watch out to not leave french comments ;)
Are you sure that 424.95 is the expected value then? If they are different, I would rely more on ref than new!

Reply:
The biorbd_ref does not let me go through with " EXPECT_NEAR(hillType.force(emg), 424.95358302550062, requiredPrecision);". I added the same new tests, but it always says that I need to compute the geometry. When I run it in biorbd_new, it works and everything passes.

Reply:
ok, I am not sure why you have that problem, but I validated the number with Release 1.1.3. So that is okay for me :)

You can resolve!